TestAssets/TestProjects/TestProjectWithCultureSpecificResource/NuGet.Config

```diff
@@ -3,7 +3,7 @@
   <packageSources>
     <!--To inherit the global NuGet package sources remove the <clear/> line below -->
     <clear />
-    <add key="dotnet-core" value="https://dotnet.myget.org/F/dotnet-core/api/v3/index.json" />
+    <add key="dotnet-core" value="https://www.myget.org/F/dotnet-core/api/v3/index.json" />
```

Comment:
This shouldn't be changed.  'dotnet.myget.org' is the new feed and the 'www.myget.org' version will be removed.